### PR TITLE
Nickquednow fix b

### PR DESCRIPTION
**Pull request by nickquednow**
would like to be approved

### CODE REVIEW

**Comment by Bobobalink** [comment]
Hey I'm pretty sure I gave you push access, so you can directly merge things yourself if you want. I didn't actually mean to make you unable to directly access these repos, it was just a default config.

I'm happy to be a second set of eyes though, if you want to keep going through this process.

File: src/main/java/frc/robot/HwPneumatics.java [hunk 1]
```diff
@@ -5,7 +5,7 @@
 import edu.wpi.first.wpilibj.Compressor;
 
 public class HwPneumatics {
-    public Compressor c = new Compressor(0);
+    public Compressor c = new Compressor(0);//needs this so we can actually use pneumatics
```
Comment (Bobobalink) [inline src/main/java/frc/robot/HwPneumatics.java]: I am reasonably certain that you do not actually need this to run the compressor, it should just run automatically. You only need this if we want to manually control when the compressor does/does not run.

File: src/main/java/frc/robot/HwMotor.java [hunk 2]
```diff
@@ -64,29 +64,6 @@
...
-    private NetworkTableEntry drivePrekI = table.getEntry("drivekI");
-    private NetworkTableEntry drivePrekD = table.getEntry("drivekD");
-    private NetworkTableEntry drivePrekF = table.getEntry("drivekF");
-    public NetworkTableEntry rightDriveVel = table.getEntry("rightDriveVelocity");
-    public NetworkTableEntry rightDriveOutput = table.getEntry("rightDriveOutput");
-    public NetworkTableEntry rightDriveVelSetpoint = table.getEntry("rightSetpointReadout");
-    public NetworkTableEntry leftDriveVel = table.getEntry("leftDriveVelocity");
-    public NetworkTableEntry leftDriveOutput = table.getEntry("leftDriveOutput");
-    public NetworkTableEntry leftDriveVelSetpoint = table.getEntry("leftSetpointReadout");
     public NetworkTableEntry ball1 = table.getEntry("BallAStatus");
```
Comment (Bobobalink) [inline src/main/java/frc/robot/HwMotor.java]: You missed getting rid of these NetworkTableEntry after you moved them into Table.java

File: src/main/java/frc/robot/HwMotor.java [hunk 3]
```diff
@@ -334,31 +311,28 @@ public void valueChanged(NetworkTable table, String key, NetworkTableEntry entry
             public void valueChanged(NetworkTable table, String key, NetworkTableEntry entry, NetworkTableValue value, int flags) {
                 double TargetVeloc = value.getDouble();
                 robot.teleop.rpm = TargetVeloc;
-                System.out.println("new target velocity: " + TargetVeloc);
             }
             
         }, EntryListenerFlags.kUpdate | EntryListenerFlags.kImmediate | EntryListenerFlags.kNew);
     }
```
Comment (Bobobalink) [inline src/main/java/frc/robot/HwMotor.java]: You put all of these in `Tables` but never removed them from `HWMotor`.

**Comment by Bobobalink** [comment]
It appears that you managed to create an empty commit with the message "orgnaized teleop... somewhat". I'm assuming you meant for this to have changes in it?

To be completely honest, I didn't know it was _possible_ to create an empty git commit, so congratulations are in order. You should just be able to make a new commit that actually has the changes you meant to commit.

**Comment by nickquednow** [comment]
I was uploading the files onto the repository... it did not process correctly and that happened XD (the empty commit). though I do agree with you on leaving the camera network tables with the camera now that you mention it... I was frustrated when looking through hwmotor and trying to find something that I needed but finding garbage tons of network tables. I had did a pull request because I did want a second pair of eyes for the sake of completion.
I will work on converting over just a little more today. If there are any more projects that might need to be done. I can work on them through GitHub. No guarantees they will work, but I can give it a shot.
I have been able to merge pull requests by the way just wanted you to see it first before I did anything.

**Pull request merged**